web/puppeteer/example.js

```diff
@@ -0,0 +1,10 @@
+const puppeteer = require('puppeteer');
+
```

Comment:
Can we change the file extension from `js` to `ts`, so the whole package is written in TypeScript?